### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by egillax**
- skip tests that require github secrets
- Document univariate and stepwise selection, was causing a R cmd check warning

### CODE REVIEW

**Comment by egillax** [comment]
@rossdwilliams can you review? With this PR the tests pass.

File: R/VariableSelection.R [hunk 1]
```diff
@@ -93,6 +98,14 @@ fitUnivariateSelection <- function(
...
+#'
+#' @param modelSettings settings of model to use in fit after selecting variables
+#' @param selectMethod `backward` or `forward` selection
+#' @param nInitialVariables # of variables to select initially
+#' @param nVariables amount of variables to select, default `20`
+#' @param stepSize  How many variables to add/remove in each step
+#'
 #' @export
 setStepwiseSelection <- function(modelSettings,
                                  selectMethod = "backward",
```
Comment (rossdwilliams) [inline R/VariableSelection.R:114]: as this is either forward or backward, can you add a check for that?

File: R/VariableSelection.R [hunk 2]
```diff
@@ -16,7 +16,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
+#' Selects features based on univariate statistics
+#' 
+#' @param corMethod which type of correlation to use, default `pearson`
```
Comment (rossdwilliams) [inline R/VariableSelection.R]: what are the other available options?

**Review by rossdwilliams — CHANGES_REQUESTED** [review]
can you address the two minor comments then this is good to og

**Comment by egillax** [comment]
Comments have been addressed. Merging.

**Pull request merged**